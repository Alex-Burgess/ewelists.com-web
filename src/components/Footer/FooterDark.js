import React from "react";
import { Link } from "react-router-dom";
// @material-ui/core components
import { makeStyles } from "@material-ui/core/styles";
// @material-ui/icons
// core components
import Footer from "components/Footer/Footer.js";
import Button from "components/Buttons/Button.js";
import GridContainer from "components/Grid/GridContainer.js";
import GridItem from "components/Grid/GridItem.js";

import styles from "assets/jss/material-kit-pro-react/components/footer/footerDarkStyle.js";
const useStyles = makeStyles(styles);

export default function FooterLarge(props) {
  const classes = useStyles();

  return (
    <div>
      <Footer
        theme="dark"
        content={
          <div className={classes.right}>
            &copy; {1900 + new Date().getYear()} ewelists
          </div>
        }
      >
      <div className={classes.footer}>
        <GridContainer>
          <GridItem xs={12} sm={3} md={8}>
            <a href="#pablo">
              <h5>Join Us</h5>
            </a>
            <p>
              Join us and other parents who believe in making parenting simpler.
            </p>
            <ul className={classes.socialButtons}>
              <li>
                <Button justIcon simple href="https://www.facebook.com/ewelists" target="_blank" color="facebook">
                  <i className="fab fa-facebook-square" />
                </Button>
              </li>
              <li>
                <Button justIcon simple href="https://www.instagram.com/ewelists/" target="_blank" color="instagram">
                  <i className="fab fa-instagram" />
                </Button>
              </li>
            </ul>
          </GridItem>
          <GridItem xs={12} sm={2} md={2}>
            <h5>About</h5>
            <ul>
              <li>
                <Link to="/list-ideas">Blog</Link>
              </li>
              <li>
                <Link to="/about">About us</Link>
              </li>
              <li>
                <Link to="/contact">Contact us</Link>
              </li>
            </ul>
          </GridItem>
          <GridItem xs={12} sm={2} md={2}>
            <h5>Legal</h5>
            <ul>
              <li>
                <Link to="/privacy">Privacy</Link>
              </li>
              <li>
                <Link to="/terms">Terms</Link>
              </li>
              <li>
                <Link to="/cookies">Cookies</Link>
              </li>
            </ul>
          </GridItem>
        </GridContainer>
      </div>
      </Footer>
    </div>
  );
}
