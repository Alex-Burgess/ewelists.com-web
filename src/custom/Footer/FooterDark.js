import React from "react";
// @material-ui/core components
import { makeStyles } from "@material-ui/core/styles";
import List from "@material-ui/core/List";
import ListItem from "@material-ui/core/ListItem";
// @material-ui/icons
// core components
import Footer from "components/Footer/Footer.js";
import Button from "components/CustomButtons/Button.js";
import GridContainer from "components/Grid/GridContainer.js";
import GridItem from "components/Grid/GridItem.js";

import styles from "assets/jss/custom/components/footerDarkStyle.js";
const useStyles = makeStyles(styles);

export default function FooterLarge(props) {
  const classes = useStyles();

  return (
    <div>
      <Footer
        theme="dark"
        content={
          <div>
            <div className={classes.left}>
              <List className={classes.list}>
                <ListItem className={classes.inlineBlock}>
                  <a
                    href="/contact"
                    className={classes.block}
                  >
                    Contact Us
                  </a>
                </ListItem>
                <ListItem className={classes.inlineBlock}>
                  <a
                    href="/#about"
                    className={classes.block}
                  >
                    About us
                  </a>
                </ListItem>
                <ListItem className={classes.inlineBlock}>
                  <a
                    href="/privacy"
                    className={classes.block}
                  >
                    Privacy
                  </a>
                </ListItem>
                <ListItem className={classes.inlineBlock}>
                  <a
                    href="/terms"
                    className={classes.block}
                  >
                    Terms
                  </a>
                </ListItem>
              </List>
            </div>
            <div className={classes.right}>
              &copy; {1900 + new Date().getYear()} ewelists
            </div>
          </div>
        }
      >
      <GridContainer>
        <GridItem xs={12} sm={8} md={8}>
          <h4 id="about">About Us</h4>
          <p>
            Ewelists is an online gift list tool, focused on helping new and expectant parents.
            Our goal is simple, to make creating a gift list for your little one's special occasion, such as baby shower,
            first birthday or Christmas faster and easier.{" "}
          </p>
          <p>
            We started by building the tool we would want to use, but would also love to hear your ideas.
            If you have any suggestions please contact us!{" "}
          </p>
        </GridItem>
        <GridItem xs={12} sm={4} md={4}>
          <h4>Follow Us</h4>
          <ul className={classes.socialButtons}>
            <li>
              <Button justIcon simple href="https://www.facebook.com/ewelists" target="_blank" color="facebook">
                <i className="fab fa-facebook-square" />
              </Button>
            </li>
            <li>
              <Button justIcon simple href="https://twitter.com/ewelists" target="_blank" color="twitter">
                <i className="fab fa-twitter" />
              </Button>
            </li>
            <li>
              <Button justIcon simple href="https://www.instagram.com/ewelists/" target="_blank" color="instagram">
                <i className="fab fa-instagram" />
              </Button>
            </li>
          </ul>
        </GridItem>
      </GridContainer>
      </Footer>
    </div>
  );
}
