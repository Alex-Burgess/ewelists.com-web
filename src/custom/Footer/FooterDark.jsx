/*!

=========================================================
* Material Kit PRO React - v1.7.0
=========================================================

* Product Page: https://www.creative-tim.com/product/material-kit-pro-react
* Copyright 2019 Creative Tim (https://www.creative-tim.com)

* Coded by Creative Tim

=========================================================

* The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.

*/
/*eslint-disable*/
import React from "react";
// nodejs library to set properties for components
import PropTypes from "prop-types";
// nodejs library that concatenates classes
import classNames from "classnames";
// @material-ui/core components
import withStyles from "@material-ui/core/styles/withStyles";
import InputAdornment from "@material-ui/core/InputAdornment";
import List from "@material-ui/core/List";
import ListItem from "@material-ui/core/ListItem";
import Icon from "@material-ui/core/Icon";
// @material-ui/icons
import Face from "@material-ui/icons/Face";
// core components
import Footer from "components/Footer/Footer.jsx";
import Button from "components/CustomButtons/Button.jsx";
import GridContainer from "components/Grid/GridContainer.jsx";
import GridItem from "components/Grid/GridItem.jsx";

import footerLargeStyle from "assets/jss/custom/components/footerDarkStyle.jsx";

function FooterLarge(props) {
  const { classes } = props;

  return (
    <div>
      <Footer
        theme="dark"
        content={
          <div>
            <div className={classes.left}>
              <List className={classes.list}>
                <ListItem className={classes.inlineBlock}>
                  <a
                    href="/contact"
                    className={classes.block}
                  >
                    Contact Us
                  </a>
                </ListItem>
                <ListItem className={classes.inlineBlock}>
                  <a
                    href="/#about"
                    className={classes.block}
                  >
                    About us
                  </a>
                </ListItem>
                <ListItem className={classes.inlineBlock}>
                  <a
                    href="/privacy"
                    className={classes.block}
                  >
                    Privacy
                  </a>
                </ListItem>
                <ListItem className={classes.inlineBlock}>
                  <a
                    href="/terms"
                    className={classes.block}
                  >
                    Terms
                  </a>
                </ListItem>
              </List>
            </div>
            <div className={classes.right}>
              &copy; {1900 + new Date().getYear()} ewelists
            </div>
          </div>
        }
      >
      <GridContainer>
        <GridItem xs={12} sm={8} md={8}>
          <h4 id="about">About Us</h4>
          <p>
            Ewelists is an online gift list tool, focused on helping new and expectant parents.
            Our goal is simple, to make creating a gift list for your little one's special occasion, such as baby shower,
            first birthday or Christmas faster and easier.{" "}
          </p>
          <p>
            We started by building the tool we would want to use, but would also love to hear your ideas.
            If you have any suggestions please contact us!{" "}
          </p>
        </GridItem>
        <GridItem xs={12} sm={4} md={4}>
          <h4>Follow Us</h4>
          <ul className={classes.socialButtons}>
            <li>
              <Button justIcon simple href="https://www.facebook.com/ewelists" target="_blank" color="facebook">
                <i className="fab fa-facebook-square" />
              </Button>
            </li>
            <li>
              <Button justIcon simple href="https://twitter.com/ewelists" target="_blank" color="twitter">
                <i className="fab fa-twitter" />
              </Button>
            </li>
            <li>
              <Button justIcon simple href="https://www.instagram.com/ewelists/" target="_blank" color="instagram">
                <i className="fab fa-instagram" />
              </Button>
            </li>
          </ul>
        </GridItem>
      </GridContainer>
      </Footer>
    </div>
  );
}

FooterLarge.propTypes = {
  classes: PropTypes.object
};

export default withStyles(footerLargeStyle)(FooterLarge);
