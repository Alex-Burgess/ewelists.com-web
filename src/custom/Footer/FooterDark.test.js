import * as React from 'react';
import renderer from 'react-test-renderer';
import FooterDark from './FooterDark';

test('Renders FootDark component.', () => {
  const tree = renderer
    .create(
      <FooterDark />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
