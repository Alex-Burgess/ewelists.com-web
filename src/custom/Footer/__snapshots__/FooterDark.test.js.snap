// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders FootDark component. 1`] = `
<div>
  <footer
    className="makeStyles-footer-10 makeStyles-dark-14 makeStyles-big-11"
  >
    <div
      className="makeStyles-container-16"
    >
      <div>
        <div
          className="makeStyles-content-12"
        >
          <div
            className="MuiGrid-root makeStyles-grid-27  MuiGrid-container"
          >
            <div
              className="MuiGrid-root makeStyles-grid-131  MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-8 MuiGrid-grid-md-8"
            >
              <h4
                id="about"
              >
                About Us
              </h4>
              <p>
                Ewelists is an online gift list tool, focused on helping new and expectant parents. Our goal is simple, to make creating a gift list for your little one's special occasion, such as baby shower, first birthday or Christmas faster and easier.
                 
              </p>
              <p>
                We started by building the tool we would want to use, but would also love to hear your ideas. If you have any suggestions please contact us!
                 
              </p>
            </div>
            <div
              className="MuiGrid-root makeStyles-grid-131  MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-4 MuiGrid-grid-md-4"
            >
              <h4>
                Follow Us
              </h4>
              <ul
                className="makeStyles-socialButtons-6"
              >
                <li>
                  <a
                    aria-disabled={false}
                    className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-button-132 makeStyles-facebook-143 makeStyles-simple-154 makeStyles-justIcon-162"
                    href="https://www.facebook.com/ewelists"
                    onBlur={[Function]}
                    onDragLeave={[Function]}
                    onFocus={[Function]}
                    onKeyDown={[Function]}
                    onKeyUp={[Function]}
                    onMouseDown={[Function]}
                    onMouseLeave={[Function]}
                    onMouseUp={[Function]}
                    onTouchEnd={[Function]}
                    onTouchMove={[Function]}
                    onTouchStart={[Function]}
                    tabIndex={0}
                    target="_blank"
                  >
                    <span
                      className="MuiButton-label"
                    >
                      <i
                        className="fab fa-facebook-square"
                      />
                    </span>
                  </a>
                </li>
                <li>
                  <a
                    aria-disabled={false}
                    className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-button-132 makeStyles-twitter-142 makeStyles-simple-154 makeStyles-justIcon-162"
                    href="https://twitter.com/ewelists"
                    onBlur={[Function]}
                    onDragLeave={[Function]}
                    onFocus={[Function]}
                    onKeyDown={[Function]}
                    onKeyUp={[Function]}
                    onMouseDown={[Function]}
                    onMouseLeave={[Function]}
                    onMouseUp={[Function]}
                    onTouchEnd={[Function]}
                    onTouchMove={[Function]}
                    onTouchStart={[Function]}
                    tabIndex={0}
                    target="_blank"
                  >
                    <span
                      className="MuiButton-label"
                    >
                      <i
                        className="fab fa-twitter"
                      />
                    </span>
                  </a>
                </li>
                <li>
                  <a
                    aria-disabled={false}
                    className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-button-132 makeStyles-instagram-153 makeStyles-simple-154 makeStyles-justIcon-162"
                    href="https://www.instagram.com/ewelists/"
                    onBlur={[Function]}
                    onDragLeave={[Function]}
                    onFocus={[Function]}
                    onKeyDown={[Function]}
                    onKeyUp={[Function]}
                    onMouseDown={[Function]}
                    onMouseLeave={[Function]}
                    onMouseUp={[Function]}
                    onTouchEnd={[Function]}
                    onTouchMove={[Function]}
                    onTouchStart={[Function]}
                    tabIndex={0}
                    target="_blank"
                  >
                    <span
                      className="MuiButton-label"
                    >
                      <i
                        className="fab fa-instagram"
                      />
                    </span>
                  </a>
                </li>
              </ul>
            </div>
          </div>
        </div>
        <hr />
      </div>
      <div>
        <div
          className="makeStyles-left-4"
        >
          <ul
            className="MuiList-root makeStyles-list-3 MuiList-padding"
          >
            <li
              className="MuiListItem-root makeStyles-inlineBlock-2 MuiListItem-gutters"
              disabled={false}
            >
              <a
                className="makeStyles-block-1"
                href="/contact"
              >
                Contact Us
              </a>
            </li>
            <li
              className="MuiListItem-root makeStyles-inlineBlock-2 MuiListItem-gutters"
              disabled={false}
            >
              <a
                className="makeStyles-block-1"
                href="/#about"
              >
                About us
              </a>
            </li>
            <li
              className="MuiListItem-root makeStyles-inlineBlock-2 MuiListItem-gutters"
              disabled={false}
            >
              <a
                className="makeStyles-block-1"
                href="/privacy"
              >
                Privacy
              </a>
            </li>
            <li
              className="MuiListItem-root makeStyles-inlineBlock-2 MuiListItem-gutters"
              disabled={false}
            >
              <a
                className="makeStyles-block-1"
                href="/terms"
              >
                Terms
              </a>
            </li>
          </ul>
        </div>
        <div
          className="makeStyles-right-5"
        >
          Â© 
          2020
           ewelists
        </div>
      </div>
      <div
        className="makeStyles-clearFix-26"
      />
    </div>
  </footer>
</div>
`;
